### PR TITLE
Bugfix/209 export specobject title

### PR DESCRIPTION
**Pull request by kaklakariada**
This implements #209

### CODE REVIEW

File: src/test/java/org/itsallcode/openfasttrace/exporter/specobject/TestSpecobjectExportImport.java [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+    {
+        final SpecificationItem itemA = SpecificationItem.builder() //
+                .id(SpecificationItemId.createId("foo", "bar", 1)) //
+                .status(ItemStatus.PROPOSED) //
+                .description("the description") //
+                .rationale("the rationale") //
+                .comment("the comment") //
+                .location(Location.create("dummy.xml", 4)) //
+                .build();
+        final SpecificationItem itemB = SpecificationItem.builder() //
```
Comment (redcatbear) [inline src/test/java/org/itsallcode/openfasttrace/exporter/specobject/TestSpecobjectExportImport.java]: Tests too much in one test (e.g. multi-line behavior).

File: src/test/java/org/itsallcode/openfasttrace/exporter/specobject/TestSpecobjectExportImport.java [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+
+    private List<SpecificationItem> importItems(final String exportedItems)
+    {
+        final InputFile input = StreamInput.forContent(Paths.get("dummy.xml"), exportedItems);
+        final SpecificationListBuilder itemBuilder = SpecificationListBuilder.create();
+        new SpecobjectImporterFactory().createImporter(input, itemBuilder).runImport();
+        return itemBuilder.build();
+    }
+
+    private String exportToString(final SpecificationItem... items)
```
Comment (redcatbear) [inline src/test/java/org/itsallcode/openfasttrace/exporter/specobject/TestSpecobjectExportImport.java]: Alternatively use the System Out Extension to make test more compact and remove potential duplication with other tests.

File: src/test/java/org/itsallcode/openfasttrace/exporter/specobject/TestSpecobjectExportImport.java [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+                .rationale("another\nrationale") //
+                .comment("another\ncomment") //
+                .location(Location.create("dummy.xml", 5)) //
+                .build();
+        assertExportAndImport(itemA, itemB);
+    }
+
+    private void assertExportAndImport(final SpecificationItem... items)
+    {
+        final String exportedItems = exportToString(items);
```
[the remark this answers is not in the document]
Comment (kaklakariada) [reply src/test/java/org/itsallcode/openfasttrace/exporter/specobject/TestSpecobjectExportImport.java]: The goal of this is class is to verify that import and export are compatible. String comparison is used in separate tests for importer and exporter

File: src/test/java/org/itsallcode/openfasttrace/exporter/specobject/TestSpecobjectExportImport.java (hunk 2 above)
Comment (kaklakariada) [reply src/test/java/org/itsallcode/openfasttrace/exporter/specobject/TestSpecobjectExportImport.java]: In my opinion extensions like this should only be used if there is no other solution.

**Pull request merged**